app/views/pages/_form.html.erb

```diff
@@ -13,9 +13,8 @@
 
   <% if FeatureService.enabled? :detailed_guidance_enabled %>
     <h2 class="govuk-heading-m"><%= t("guidance.guidance") %></h2>
-    <% if page_object.page_heading.present? && page_object.guidance_markdown.present? %>
+    <% if (page_object.respond_to?(:page_heading) && page_object.page_heading.present?) && (page_object.respond_to?(:page_heading) && page_object&.guidance_markdown&.present?) %>
```

Comment:
Should this be `guidance_markdown` the second time? Also, I think the extra ampersands here are unnecessary?

```suggestion
    <% if (page_object.respond_to?(:page_heading) && page_object.page_heading.present?) && (page_object.respond_to?(:guidance_markdown) && page_object.guidance_markdown.present?) %>
```

Reply:
Good spot, thanks @lfdebrux.